### PR TITLE
Bug fix storage variable

### PR DESCRIPTION
**Pull request by noahrhodes**
Bug fix for #15.  run_mld_strg_uc test values have been updated.  Two tests have been added for run_mld_strg_uc with only storage elements.

### CODE REVIEW

File: test/data/case5_mld_strgonly.m [hunk 1]
```diff
@@ -0,0 +1,59 @@
+% used in tests of,
+% - storage modeling
```
Comment (ccoffrin) [inline test/data/case5_mld_strgonly.m:2]: This comment about what this is designed to test should be more specific.

File: test/data/case5_mld_strgonly.m [hunk 2]
```diff
@@ -0,0 +1,59 @@
+% used in tests of,
```
Comment (ccoffrin) [inline test/data/case5_mld_strgonly.m:1]: This file name should be `case5_mld_strg_only.m`.  Also why "only"?

File: test/runtests.jl [hunk 3]
```diff
@@ -30,6 +30,7 @@ case3_mld_lc = PowerModels.parse_file("../test/data/case3_mld_lc.m")
 case5_mld_ft = PowerModels.parse_file("../test/data/case5_mld_ft.m")
 case5_mld_strg = PowerModels.parse_file("../test/data/case5_mld_strg.m")
 case5_mld_strg_uc = PowerModels.parse_file("../test/data/case5_mld_strg_uc.m")
+case5_mld_strgonly = PowerModels.parse_file("../test/data/case5_mld_strgonly.m")
```
Comment (ccoffrin) [inline test/runtests.jl:33]: Same note as above, `case5_mld_strg_only`

**Review by ccoffrin — CHANGES_REQUESTED** [review]
A few small requests.

File: test/data/case5_mld_strgonly.m (hunk 2 above)
Comment (noahrhodes) [reply test/data/case5_mld_strgonly.m:1]: this model has not generators, only storage.  What is an appropriate name?

Comment (ccoffrin) [reply test/data/case5_mld_strgonly.m:1]: I didn't notice that the status was off for all of the generators.  This name is good.

**Comment by noahrhodes** [comment]
Naming for strgonly has been updated.  Any other fixes?
@ccoffrin

**Review by ccoffrin — APPROVED** [review]

**Comment by ccoffrin** [comment]
LGTM, we just need to wait for a tag of PowerModels and update the version requirements in this package.  Once the tests are passing, we will merge this.

**Comment by noahrhodes** [comment]
@ccoffrin 

I made some updates for switching to PowerModels v0.12.2, after the check is finished it should be ready to merge in.

The commits got a little messy because I wound up working on an old branch that I later merged in.  I'm pretty I caught all of the issues, but it might be good to double-check the code again before merging.

**Comment by ccoffrin** [comment]
The project toml deps needs to be updated `PowerModels = "~0.12"` should be changed to `PowerModels = "~0.12.2"` becouse this code now request at least version v0.12.2.

Other than that, LGTM.  

@pseudocubic do you see any reason not to merge this into master?

**Comment by pseudocubic** [comment]
Once the Project.toml is updated and Manifest.toml regenerated (`] up`) this looks fine to merge to me

**Comment by noahrhodes** [comment]
@ccoffrin @pseudocubic requirements have been updated.

**Pull request merged**